projects/person-search-index-from-delius/build.gradle.kts

```diff
@@ -6,4 +6,12 @@ tasks {
         tag("latest", "ghcr.io/ministryofjustice/hmpps-probation-integration-services/${project.name}:latest")
         files("container")
     }
+    named("docker") {
+        doLast {
+            val dir = File("${project.parent?.projectDir}/changed")
```

Comment:
Would this work for nested projects?  E.g. for the person-search management image

Reply:
Could you use `${project.rootDir}/changed` ?